cli/src/main/java/org/jboss/as/cli/gui/JConsoleCLIPlugin.java

```diff
@@ -119,6 +119,7 @@ private void connect() throws Exception {
         if (mbeanServerConn instanceof RemotingMBeanServerConnection) {
             final CommandContext cmdCtx = CommandContextFactory.getInstance().newCommandContext();
             isConnected = connectUsingRemoting(cmdCtx, (RemotingMBeanServerConnection)mbeanServerConn);
+            init(cmdCtx);
```

Comment:
It doesn't seem right to ignore the result of L121 here. The init(cmdCtx) call will end up setting isConnected = true, but if L121 returns 'false' the plugin isn't actually connected.

I haven't looked carefully, but it seems like it should be more like this:

```
if (connectUsingRemoting(cmdCtx, (RemotingMBeanServerConnection)mbeanServerConn)) {
    init(cmdCtx);
} else {
   doSomethingButBrianIsntSureWHAT();
}
```

The doSomethingButBrianIsntSureWHAT is probably display some sort of error dialog. This would mean the user has a RemotingMBeanServerConnection to something but a management channel can't be opened on it. This seems highly unlikely and the right thing to do to recover is unclear, so just failing seems reasonable.